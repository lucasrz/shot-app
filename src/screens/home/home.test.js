import React from 'react';
import renderer from 'react-test-renderer';
import Home from './index';

import store from '../../store';

it('Testing Home Component', () => {
  const tree = renderer
    .create(<Home store={store} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
