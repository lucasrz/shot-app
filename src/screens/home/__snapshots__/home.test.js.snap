// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home Component 1`] = `
<div
  className="container"
>
  <h1>
    Loading...
  </h1>
</div>
`;
